docs/modules/reference/pages/daemons/daemon-config-files/enlinkd.adoc

```diff
@@ -14,3 +32,76 @@ The enhanced link daemon collects xref:operation:topology/enlinkd/introduction.a
 | enlinkd.log
 | Logs relating to enlinkd server events.
 |===
+
+
+== Enlinkd configuration
+
+If multiple protocols are enabled, links will be discovered for each enabled discovery protocol.
+The topology web UI visualizes links for each discovery protocol.
+For example, if you start CDP and LLDP discovery, the web UI visualizes a CDP link and an LLDP link.
+
+Restart {page-component-title} or just the enlinkd daemon to apply configuration changes.
+
+.enlinkd-configuration parameters
+[options="header"]
+[cols="1,3,1"]
+|===
+| Attribute
+| Description
+| Default
+
+3+| *Integer*
+
+| threads
+| Number of parallel threads collectors, updaters, and discovery use.
```

Comment:
```suggestion
| Number of parallel thread collectors, updaters, and discovery use.
```
"threads" sounds wrong to me, but if it's common use, ignore my comment.

Reply:
It's the number of threads used by collectors, updaters, and discovery. I'll tweak the wording to clarify.

Reply:
```suggestion
| Number of parallel threads for enlinkd collector, updater, and discovery tasks.
```